model/Hierarchy.php

```diff
@@ -218,10 +218,12 @@ public function getChildrenAsUL($attributes = "", $titleEval = '"<li>" . $child-
 	 * @param int $nodeCountThreshold See {@link getChildrenAsUL()}
 	 * @return int The actual number of nodes marked.
 	 */
-	public function markPartialTree($nodeCountThreshold = 30, $context = null,
+	public function markPartialTree($nodeCountThreshold = null, $context = null,
 			$childrenMethod = "AllChildrenIncludingDeleted", $numChildrenMethod = "numChildren") {
 
-		if(!is_numeric($nodeCountThreshold)) $nodeCountThreshold = 30;
+		if(!is_numeric($nodeCountThreshold)) {
+			$nodeCountThreshold = Config::inst()->get('Hierarchy', 'node_threshold_total');
```

Comment:
I would love to have done, but `Config::inst()->get('Hierarchy', 'node_threshold_total')` is used elsewhere in the same class. If it were to be included, it would be an unrelated change being included in this PR. Are you happy for other refs to it to be changed also?